packages/adaptive-ui-designer-figma/src/cli/figma-rest-client.ts

```diff
@@ -0,0 +1,505 @@
+import type * as FigmaRestAPI from "@figma/rest-api-spec";
```

Comment:
Is there nowhere we can reference this instead? Maybe add a link to where it came from in the header?

Reply:
Sorry, I don't follow you. What is the issue?

Reply:
Did you write this whole file or is it a sample or something Figma provided? Seems like a lot that we have to maintain, like it should be something they publish.

Reply:
Oh, yeah I wrote this all.  I was also surprised they didn't just have a lib to do this. There are a few others around but the ones I found didn't use the latest published types.